src/utils/awsAccess.ts

```diff
@@ -0,0 +1,96 @@
+import jwt from "@tsndr/cloudflare-worker-jwt";
+import { v4 as uuidv4 } from "uuid";
+import { env } from "../typeDefinitions/default.types";
+import config from "../../config/config";
+import { discordTextResponse } from "./discordResponse";
+import { DISCORD_BASE_URL, AWS_IAM_SIGNIN_URL } from "../constants/urls";
+
+export async function processAWSAccessRequest(
+  discordUserId: string,
+  awsGroupId: string,
+  env: env,
+  TraceId: string,
+  channelId: number
+) {
+  const authToken = await jwt.sign(
```

Comment:
Don't we already have this? why write it again?

Reply:
didn't get this comment, we are using this in similar manner in one of the other API too.

Reply:
So at every place we are manually signing, So don't already have a common thing to do this, if not then please create one?

Reply:
Have created a common utility for this.